src/main/java/com/minecolonies/coremod/blocks/BlockHutBaker.java

```diff
@@ -1,5 +1,7 @@
 package com.minecolonies.coremod.blocks;
 
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.World;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.world.World'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)